src/libraries/System.Net.HttpListener/src/System/Net/Managed/ListenerAsyncResult.Managed.cs

```diff
@@ -79,17 +79,17 @@ internal void Complete(Exception exc)
         }
 
         private static WaitCallback s_invokeCB = new WaitCallback(InvokeCallback);
-        private static void InvokeCallback(object o)
+        private static void InvokeCallback(object? o)
```

Comment:
The argument is nullable but we allow null dereference later by `!`. In this case I would propose make the argument non nullable and use `!` in the `InvokeCallback` invocation (`InvokeCallback(o!)`). It will highlight problematic usage of this method.

Reply:
That's what I had first :) But then, `InvokeCallback` is a delegate of type `WaitCallback` that takes in an `object?` parameter, so the compiler would complain. I tried `InvokeCallback!` though and that worked, so I ended up making this change :)